compiler/one-cmds/tests/onecc_044.test

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
+
+# Copyright (c) 2023 Samsung Electronics Co., Ltd. All Rights Reserved
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Test for "O1=True" option with other options
+
+filename_ext="$(basename -- $0)"
+filename="${filename_ext%.*}"
+
+trap_err_onexit()
+{
+  echo "${filename_ext} FAILED"
+  exit 255
+}
+
+trap trap_err_onexit ERR
+
+configfile="onecc_044.cfg"
+outputfile="test_onnx_model.onecc_044.opt.circle"
+
+rm -f ${filename}.log
+rm -rf ${outputfile}
+
+# run test
+onecc -C ${configfile} > ${filename}.log 2>&1
+
+if [[ ! -s "${outputfile}" ]]; then
+  trap_err_onexit
+fi
+
+# Check convert_nchw_to_nhwc is included
+if ! grep -q "convert_nchw_to_nhwc" <<< $(cat ${outputfile}.log); then
+  trap_err_onexit
+fi
+
+# Check fold_add_v2 is excluded
+if grep -q "fold_add_v2" <<< $(cat ${outputfile}.log); then
```

Comment:
Maybe this would work, too.
```suggestion
if ! grep -q "convert_nchw_to_nhwc" ${outputfile}.log; then
  trap_err_onexit
fi

# Check fold_add_v2 is excluded
if grep -q "fold_add_v2" ${outputfile}.log; then
```